Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libc"
-version = "0.2.70"
+version = "0.2.71"
```

Comment:
I'd like to wait for a while to make a release since we recently did.

Reply:
Sure